Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -244,73 +244,62 @@ static void Setup()
 
 static bool CheckDeviceAccess(libusb_device* device)
 {
-  int ret;
   libusb_device_descriptor desc;
-  int dRet = libusb_get_device_descriptor(device, &desc);
-  if (dRet)
+  int ret = libusb_get_device_descriptor(device, &desc);
+  if (ret)
   {
     // could not acquire the descriptor, no point in trying to use it.
-    ERROR_LOG(SERIALINTERFACE, "libusb_get_device_descriptor failed with error: %d", dRet);
+    ERROR_LOG(SERIALINTERFACE, "libusb_get_device_descriptor failed with error: %d", ret);
     return false;
   }
 
-  if (desc.idVendor == 0x057e && desc.idProduct == 0x0337)
+  if (desc.idVendor != 0x057e || desc.idProduct != 0x0337)
+    // This isn’t the device we are looking for.
```

Comment:
This should be braced, since the contents of the body travels more than one line. LGTM otherwise.

Reply:
This is now fixed, thanks.